### DISCUSSION TITLE
Where is the database structure json file (e.g. document and collection)?

### DISCUSSION THREAD

**Discussion by afokapu** [TheAlphamerc/flutter_twitter_clone#250] [Q&A] [answered]
@TheAlphamerc  thank you !
One question as I am a beginner with firestore / firebase
Where is the database structure json file (e.g. document and collection)?

**Comment by TheAlphamerc** [comment, marked as answer]
@afokapu You can find your answer to the query here -> https://github.com/TheAlphamerc/flutter_twitter_clone/issues/171#issuecomment-841787483

**Reply by afokapu** [reply]
Thanks !
